noxfile.py

```diff
@@ -32,19 +45,49 @@ def default(session):
     session.install("mock", "pytest", "pytest-cov", "grpcio >= 1.0.2")
     session.install("-e", ".")
 
-    # Run py.test against the unit tests.
-    session.run(
-        "py.test",
-        "--quiet",
-        "--cov=google.api_core",
-        "--cov=tests.unit",
-        "--cov-append",
-        "--cov-config=.coveragerc",
-        "--cov-report=",
-        "--cov-fail-under=0",
-        os.path.join("tests", "unit"),
-        *session.posargs
-    )
+    # Inject AsyncIO content, if version >= 3.6.
+    if _greater_or_equal_than_36(session.python):
```

Comment:
Does a string comparison miss some cases?

```py
if session.python >= '3.6'
```

You can also use [packaging.version.parse](https://packaging.pypa.io/en/latest/version/#packaging.version.parse).

Reply:
I was using `packaging.version`, but Python in CI doesn't have the `packaging` package.
```
ModuleNotFoundError: No module named 'packaging'
```
https://source.cloud.google.com/results/invocations/d91e385b-d996-405c-9c94-4ba26e26abad/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-api-core%2Fpresubmit%2Fpresubmit/log

I am afraid of two corner case: 1) Python goes to 4.0; 2) Python goes to 3.10+. In second corner case, the simple string comparison fails.